docs/contributing/contributing.md

```diff
@@ -0,0 +1,99 @@
+
+### 2. Discussion
+
+Once you've written up your idea, post it to the [issue tracker](https://www.github.com/vsoch/oci-contribution/issues). You might also [post to the development list](https://groups.google.com/a/opencontainers.org/forum/#!forum/dev) to capture people's attention, and point them to your issue.  Why would you want to use the issue tracker in combination with the list? The issue tracker will allow you to keep a clean record of progress. You can easily link to code you are working on, and the discussion. It's a central place for *all the things*.
+The mailing list is important for reach and perspective. You can challenge others
+that have different use cases for interaction with containers to give feedback on your idea.
+
+### 3. Support and Action
+
+Once discussion starts to die down, it's important to try and synthesize the content discussed into a concrete action. The maintainers will likely be doing this without thinking about it. For example: 
+
+ - *Documentation Action* if there is functionality you thought was missing and it actually existed? This calls for an improvement to the documentation. The maintainer can assign issues to developers to address your concern, and ask for your feedback.
+ - *Show Me Action*: The most likely result of you wanting a new thing will be a request for a dummy implementation. It doesn't need to be perfect, but you must *show* that your idea has feet, and solves the problem that you stated. 
+ - *Feature Request Action* It could be that your idea was succinct and strong, and a new feature is called for development. This likely will result in one or more developers (including you!) coming up with a plan of action. This is less likely an option, because changes to the specifications tend to be more conservative.
+
+Generally, you will want to find support for your action, whether it be advice or actual help from others in the list. We do this step before detailed feedback in the case that your idea doesn't need furter investigation (already solved or being worked on, for example). What should never happen is anything that goes against the [code of conduct](https://github.com/opencontainers/org/blob/master/code-of-conduct.md). You should not be put down, ridiculed for your ideas, or otherwise have someone treat you poorly.
```

Comment:
nit: s/furter/further/

Reply:
I'm surprised I don't have a ton more spelling mistakes, I use gedit and it doesn't help with these things, good thing you caught this one because it looks like another word that is pretty funny :)